man/assocTestAggregate.Rd

```diff
@@ -83,12 +84,15 @@
     \item{Score.SE}{The estimated standard error of the Score}
     \item{Score.Stat}{The score Z test statistic}
     \item{Score.pval}{The score p-value}
-    If \code{burden.test} is \code{"Wald"}:
-    \item{Est}{The effect size estimate for a one unit increase in the burden value}
-    \item{Est.SE}{The estimated standard error of the effect size estimate}    
-    \item{Wald.Stat}{The Wald Z test statistic}
-    \item{Wald.pval}{The Wald p-value}
-    %% If \code{burden.test} is "Firth":
+    \item{Est.approx}{An approximation of the effect size estimate for each additional unit of burden}
```

Comment:
@smgogarten what do you think of these names? I actually don't really like "Est" anymore, but that would be consistent with what we used to return from the Wald test. I think I'd prefer something like "EffSize" or even "beta", but it might be best to keep it as similar as possible.

Reply:
We could also just make the names Est, Est.SE, and PVE. Est and Est.SE are what Wald used to return. The calculation of Est is actually identical to what was returned by Wald; Est.SE is very similar but not identical. PVE is new either way.

Reply:
I would keep the names the same as they used to be, it's minimally disruptive.